decentraland/unity-renderer
# fix: Material transition handling for AB

## What does this PR change?

Fixes the crash in scene 54,-71 for Ropsten. The effect transition was being added multiple times to objects that already had it, causing an accumulation of effects that werent necessary. 

It was solved by waiting the end of the transition before starting the OnSuccess events for loading. 

Also solves [#2745](https://github.com/decentraland/unity-renderer/issues/2745)

## How to test the changes?

Ropsten
1. Go to https://play.decentraland.zone/?renderer-branch=fix/material-transition-failling-on-fast-objects&NETWORK=ropsten&position=54,-71
2. Walk around for a little while and check that there are no hiccups or crashes

Salmonomicon

1. Go to the Solomonicon scene walking from Genesis Plaza
2. Play the TechDemo. Collect the pages and leave the head on the altar. Check that the head is visible

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
